spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/web/SpringDataWebAutoConfiguration.java

```diff
@@ -79,4 +81,9 @@ public SortHandlerMethodArgumentResolverCustomizer sortCustomizer() {
 		return (resolver) -> resolver.setSortParameter(this.properties.getSort().getSortParameter());
 	}
 
+	@Bean
+	public SpringDataWebSettings springDataWebSettings() {
```

Comment:
We should make this `@ConditionalOnMissingBean` and add a test for that behavior.

Reply:
done.